### PR TITLE
push rendered manifests into the cluster during deployment

### PR DESCRIPTION
**Pull request by addyess**

### CODE REVIEW

**Review by stonepreston — APPROVED** [review]
LGTM!

File: tests/data/bundle.yaml [hunk 1]
```diff
@@ -4,9 +4,10 @@ applications:
   kubernetes-autoscaler:
     charm: {{charm}}
     options: {{juju_args}}
+    trust: true
     scale: 1
     resources:
-      juju-autoscaler-image: kennethreitz/httpbin:latest
+      juju-autoscaler-image: rocks.canonical.com/cdk/cluster-autoscaler-juju-amd64:dev-2.9.22-build0
```
Comment (addyess) [inline tests/data/bundle.yaml:10]: @stonepreston this isn't right is it?

Comment (stonepreston) [reply tests/data/bundle.yaml:10]: ah yeah good catch

**Review by stonepreston — APPROVED** [review]

File: tests/data/bundle.yaml (hunk 1 above)
Comment (addyess) [reply tests/data/bundle.yaml:10]: @stonepreston ... i'll wait for the arch agnostic image :)

**Review by kwmonroe — APPROVED** [review]

**Pull request merged**